node/blockchain/management/commands/send_signed_change_request.py

```diff
@@ -0,0 +1,74 @@
+import json
+
+from blockchain.facade import BlockchainFacade
+from django.core.management import BaseCommand
+from requests import HTTPError
+
+from node.blockchain.inner_models import (
+    CoinTransferSignedChangeRequestMessage, Node, NodeDeclarationSignedChangeRequestMessage, SignedChangeRequest
+)
+from node.blockchain.inner_models.signed_change_request_message import CoinTransferTransaction
+from node.blockchain.types import AccountNumber, SigningKey, Type
+from node.core.clients.node import NodeClient
+from node.core.utils.cryptography import derive_public_key
+
+SIGNED_CHANGE_REQUEST_MESSAGE_MAP = {
+    Type.NODE_DECLARATION.name: NodeDeclarationSignedChangeRequestMessage,
+    Type.COIN_TRANSFER.name: CoinTransferSignedChangeRequestMessage,
+}
+
+
+class Command(BaseCommand):
+    help = 'Submit signed change requests of different types'  # noqa: A003
+
+    def add_arguments(self, parser):
+        parser.add_argument('--address', type=str, help='Network address of Node')
+        parser.add_argument('--signing-key', type=SigningKey, help='Signing (private) key')
+
+        subparsers = parser.add_subparsers(title='subcommands', dest='scr_type', required=True)
+
+        # Node Declaration
+        node_declaration_parser = subparsers.add_parser(
+            Type.NODE_DECLARATION.name, help='Node declaration signed change request'
+        )
+        node_declaration_parser.add_argument('--identifier', type=AccountNumber, required=True, help='Node identifier')
+        node_declaration_parser.add_argument(
+            '--addresses', nargs='+', type=str, required=True, help='One of many network addresses'
+        )
+        node_declaration_parser.add_argument('--fee', type=int, required=True, help='Fee amount')
+
+        # Coin Transfer
+        coin_transfer_parser = subparsers.add_parser(
+            Type.COIN_TRANSFER.name, help='Coin transfer signed change request'
+        )
+        coin_transfer_parser.add_argument(
+            '--recipient', type=AccountNumber, required=True, help='Account number of recipient'
+        )
+        coin_transfer_parser.add_argument('--amount', type=int, required=True, help='Amount of coins')
+        coin_transfer_parser.add_argument('--is_fee', type=bool, default=None, help='Is fee')
+        coin_transfer_parser.add_argument('--memo', type=bool, default=None, help='Transaction message')
+
+    def handle(self, address, signing_key, scr_type, *args, **options):
+        scr_message_class = SIGNED_CHANGE_REQUEST_MESSAGE_MAP[scr_type]
+
+        blockchain_facade = BlockchainFacade.get_instance()
+        scr_arguments = dict(account_lock=blockchain_facade.get_account_lock(derive_public_key(signing_key)))
```

Comment:
this is not correct, account lock must be take from remote blockchain, not local

Reply:
Ok, I will fix it.